specs/signature-specification.md

```diff
@@ -179,6 +152,84 @@ These attributes are considered unsigned with respect to the signing key that ge
 - **Timestamp signature** : An OPTIONAL counter signature which provides [authentic timestamp](#signing-time)e.g. Time Stamp Authority (TSA) generated timestamp signature. Only [RFC3161][ietf-rfc3161] compliant TimeStampToken are currently supported.
 - **Signing Agent**: An OPTIONAL claim that provides the identifier of the software (e.g. Notation) that produced the signature on behalf of the user. It is an opaque string set by the software that produces the signature. It's intended primarily for diagnostic and troubleshooting purposes, this attribute is unsigned, the verifier MUST NOT validate formatting, or fail validation based on the content of this claim. The suggested format is one or more tokens of the form `{id}/{version}` containing identifier and version of the software, separated by spaces. E.g. "notation/1.0.0", "notation/1.0.0 myplugin/0.8".
 
+## OCI Signatures
+
+This section describes how a Notary Project signature is stored in an OCI Distribution conformant registry.
+OCI image manifest is used to store signatures in the registry, see [OCI image spec v1.1.0-rc3][oci-image-manifest] for details.
```

Comment:
[`image-spec v1.1.0`](https://github.com/opencontainers/image-spec/releases/tag/v1.1.0) was just released!

Reply:
This is out of the scope for this PR. Will create an issue to take care of this separately (I just want to make sure we are doing it thoroughly across all the files and not just this occurrence)

Reply:
https://github.com/notaryproject/specifications/issues/295